com.unity.render-pipelines.high-definition/Documentation~/Ray-Tracing-Path-Tracing.md

```diff
@@ -53,47 +53,69 @@ Path tracing uses the [Volume](Volumes.md) framework, so to enable this feature,
 
 **Minimum Depth** set to 2, **Maximum Depth** set to 2: indirect lighting only (1 bounce)
 
-## Materials parameterization
+## Setting path tracing parameters for Materials
```

Comment:
I guess the meaning should be slightly different as we don't have specific parameters for path tracing. It is more that some parameters are ignore, or have a slight different behavior.

So maybe we should say: Material parametrization change with Path Tracer.   Or something like that.